lib/new_relic/agent/agent_helpers/connect.rb

```diff
@@ -62,10 +66,8 @@ def log_error(error)
         # no longer try to connect to the server, saving the
         # application and the server load
         def handle_license_error(error)
-          ::NewRelic::Agent.logger.error( \
-            error.message, \
-            'Visit NewRelic.com to obtain a valid license key, or to upgrade your account.'
-          )
+          ::NewRelic::Agent.logger.error(error.message,
+            'Visit NewRelic.com to obtain a valid license key, or to upgrade your account.')
```

Comment:
I think our company has changed its preferred capitalization. Not a big deal, but 🤷 
```suggestion
            'Visit newrelic.com to obtain a valid license key, or to upgrade your account.')
```